run_conformance_tests.py

```diff
@@ -81,22 +82,19 @@ def handle_args(argv):
         help='The name of the implementation to be displayed in the report.',
         type=str,
         required=True)
-    parser.add_argument(
-        '-V',
-        '--verbose',
-        help='Enable verbose CTest output',
-        type=bool,
-        default=False,
-        required=False)
```

Comment:
Where is this verbose mode gone?

Reply:
Replaced by `--additional-ctest-args`, so the caller can pass `-V` there instead